src/DistributedLock.ZooKeeper/DistributedLock.ZooKeeper.csproj

```diff
@@ -45,19 +45,19 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="ZooKeeperNetEx" Version="3.4.12.4" />
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
-    <PackageReference Include="Nullable" Version="1.3.1">
+    <PackageReference Include="ZooKeeperNetEx" />
+    <PackageReference Include="Microsoft.SourceLink.GitHub" PrivateAssets="All" />
+    <PackageReference Include="Nullable" Condition="'$(TargetFramework)' != 'netstandard2.1'">
```

Comment:
Given `<TargetFrameworks>netstandard2.0;netstandard2.1;net461</TargetFrameworks>` on line 4 of this file, I think it was an oversight that this `Condition` didn't exist. Note that this change is reflected in the lock file diff:
https://github.com/madelson/DistributedLock/pull/186/commits/c87fb79e853c7b5ab38e980ec605d711bda38c23#diff-b74c05733e0ca95b4f1c079e5455270db5118c2ba952b1cbc46d7a1452a8fb18L175. (Reminder that [netstandard2.1 uses C# 8 by default](https://devblogs.microsoft.com/dotnet/try-out-nullable-reference-types/).)

[Comment](https://github.com/madelson/DistributedLock/pull/186#discussion_r1501688868) from @madelson:
> Does it break without this condition? I would have thought the package just noops in that case.

Reply:
@madelson  If you remove Condition only from `ZooKeeper.csproj`, you get
> DistributedLock\src\DistributedLock.ZooKeeper\DistributedLock.ZooKeeper.csproj : error NU1010: The PackageReference items Nullable do not have corresponding PackageVersion. 

This makes sense because
- `ZooKeeper.csproj` targets `netstandard2.1`
- `Condition="'$(TargetFramework)' != 'netstandard2.1'"` exists for `Directory.Packages.props`
- the version must always be specified in `Directory.Packages.props`, but in this case, the version is not specified for `Nullable` in `netstandard2.1`

It's okay to install `Nullable` for `netstandard2.1` here (or in other projects), but it's unnecessary.